flasksrc/__init__.py

```diff
@@ -66,13 +66,21 @@ def create_app(test_config=None):
     api.add_resource(ToyNetSurveyById, '/api/survey/<string:survey_id>')
     docs.register(ToyNetSurveyById)
 
-    from .session import ToyNetSession, ToyNetSessionById, ToyNetSessionByIdTerminate
+    from .session import ToyNetSession, ToyNetSessionById, \
+        ToyNetSessionByIdTerminate, ToyNetSessionByIdCreateHost, \
+        ToyNetSessionByIdDeleteDevice
```

Comment:
Just so I understand the roadmap, we are going to have:
* `ToyNetSessionByIdCreateHost`
* `ToyNetSessionByIdCreateSwitch`
* `ToyNetSessionByIdCreateRouter`
*  a single `ToyNetSessionByIdDeleteDevice` which makes sure there are no links?

Reply:
Only other ones will be a `ToyNetSessionByIdCreateLink` and a `ToyNetSessionByIdDeleteLink` since they will have different parameter structures from the rest.